pkg/execution/plugins/generic/secretplugin/types.go

```diff
@@ -0,0 +1,5 @@
+package secretplugin
+
+type Spec struct {
+	Data map[string][]byte `json:"data,omitempty"`
```

Comment:
omitempty would produce an object which doesn't conform to the stated schema in secret_plugin.go